### ISSUE TITLE
`kimg_psd.dll` should prob not be included by default

### ISSUE THREAD

**Issue by brian6932** [jurplel/qView#672]
**Environment:**
- OS: Windows 10
- qView version: nightly 2023-12-27.1

**Information:**
I often go through my folders with arrow keys, and most of the time `psd` files don't actually open within qView and throw an error. Ofc I can just delete it within qView_dir/imageformats, but maybe including it by default's questionable. At the very least I feel like certain formats should just be excluded/excludable from folder seeking, so maybe this is also a feature request in disguise.
![](https://github.com/jurplel/qView/assets/18603393/ec5c2716-90b8-49dc-8771-33d6586e106d)

**Comment by jdpurcell** [comment]
Similar issues: #638, #603

**Comment by brian6932** [comment]
Ah I wasn't aware of your fork, nice

**Comment by brian6932** [comment]
A little off topic, but any idea why the newest nightlies and your fork don't theme the titlebar anymore?
I have
```pwsh
❯ $env:QT_QPA_PLATFORM
windows:darkmode=1
```
and it works on the nightly version used here which was built on Qt 6.2.2.
I don't think this is a Qt related issue as I've built programs with >= Qt 6.6.0 and they still theme the titlebar.

**Comment by jdpurcell** [comment]
I don't know, I'll have to see if I can reproduce it sometime. I'm mostly using macOS these days honestly. I know Qt has been working on [dark mode on Windows](https://www.qt.io/blog/dark-mode-on-windows-11-with-qt-6.5) and maybe they're forcing use of Fusion style to get the dark titlebar. Fusion style is the "Non-native theme" option in my fork by the way.

**Comment by brian6932** [comment]
> they're forcing use of Fusion style to get the dark titlebar

Nah, Windows can have native dark titlebars, Qt's supported this feature for a few years now.

> Fusion style is the "Non-native theme" option in my fork

Yea I noticed fusion mode, on your fork, and tried it, but wasn't able to get dark mode to work, it was just light mode, is there some separate toggle or something?

**Comment by jdpurcell** [comment]
> ... any idea why your fork (and #666) doesn't theme the titlebar anymore?

I can reproduce this - I was testing by passing the command line argument `-platform windows:darkmode=1` and got the dark title bar with Qt 6.2 but light title bar with Qt 6.5. But as I understand from Qt's blog entry and the [commit here](https://github.com/qt/qtbase/commit/5ea7e3a8111b2939f0c91b750aa1c62ab16ab715), that was an intentional change - Qt will now only use the dark title bar if the window background color is darker than the text.

> Yea I noticed fusion mode on your fork, and tried it, but wasn't able to get dark mode to work, it was just light mode, is there some separate toggle or something?

It's because of that `darkmode=1` setting apparently, `1` means just for the window frame (when allowed by the previously mentioned logic), and `2` enables full dark mode support.

**Comment by brian6932** [comment]
> `2` enables full dark mode support.

Ah yes that works, nice
